src/lsg/patterns/property-items/boolean-item/index.tsx

```diff
@@ -15,62 +15,77 @@ interface IndicatorProps {
 	checked?: boolean;
 }
 
-const StyledBooleanItem = styled.div`
+const StyledBooleanItem = styled.label`
+	display: flex;
+	align-content: center;
 	width: 100%;
+	margin-bottom: ${getSpace(SpaceSize.M)}px;
 `;
 
-const StyledLabelWrapper = styled.label`
-	display: block;
-	margin-bottom: ${getSpace(SpaceSize.L)}px;
-`;
-
-const indicatorWidth = 42;
-const indicatorHeight = 24;
-const indicatorBorderWidth = 1;
+const indicatorWidth = 48;
+const indicatorHeight = 30;
 
 const StyledIndicator = styled.span`
 	position: relative;
-	display: block;
+	display: inline-block;
 	width: ${indicatorWidth}px;
 	height: ${indicatorHeight}px;
 	border-radius: ${indicatorHeight / 2}px;
-	background: ${colors.grey90.toString()};
 	box-sizing: border-box;
-	box-shadow: inset 0 0 0 ${indicatorBorderWidth}px ${colors.grey60.toString()};
-
-	&:after {
-		content: '';
+	box-shadow: inset 0 0 0 1px ${colors.grey80.toString()};
+	transition: all ease-in-out 0.1s;
+	user-select: none;
+	&::after {
+		position: absolute;
+		content: url('data:image/svg+xml; utf8, <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><path d="M6,6L4.5,7.5L6,6L4.5,4.5L6,6z M6,6l1.5-1.5L6,6l1.5,1.5L6,6z" fill="none" stroke="${colors.grey60.toString()}" stroke-width="1" stroke-linecap="round" stroke-linejoin="round" /></svg>');
```

Comment:
We might want to move this to `lsg/patterns/icons` and create the inline string with something like this (pseudo code):

```js
const unchecked = ReactDOM.renderToString(<Icon name="x"/>);
const checked = ReactDOM.renderToString(<Icon name="ok"/>);

content: url('data:image/svg+xml,${unchecked}')
{ /* ... */ }
content: url('data:image/svg+xml,${checked}');
```